app/controllers/facility_reservations_controller.rb

```diff
@@ -178,8 +182,13 @@ def destroy
   def admin_reservation_params
     admin_params = params.require(:admin_reservation).permit(:admin_note,
                                                              :expires_mins_before,
-                                                             :category)
+                                                             :category,
+                                                             :repeats,
+                                                             :repeat_frequency,
+                                                             :repeat_end_date)
     admin_params[:expires_mins_before] = nil if params[:admin_reservation][:expires] == "0"
+    admin_params[:repeat_frequency] = nil if params[:admin_reservation][:repeats] == "0"
```

Comment:
Just an option: `ActiveRecord::Type::Boolean.new.type_cast_from_database("0") == false`

However, this moves in Rails 5.

https://gist.github.com/equivalent/3825916

Reply:
I feel like if this changes in Rails 5, not worth changing now.